README.md

```diff
@@ -30,16 +30,147 @@ MIDDLEWARE = [
 ]
 ```
 
-Finally, create a layout which exposes `{% block inertia %}{% endblock %}` in the body and set the path to this layout as `INERTIA_LAYOUT` in your `settings.py` file.
+#### Vite Installation
 
-Now you're all set!
+Before you start it's recommended to start a new app. For this example we created a app called `core`.
+
+Install the following python packages via pip
+```bash
+pip install django-vite whitenoise
```

Comment:
We don't need whitenoise, but your example uses the Django adapter for InertiaJS.
Maybe we can swap those?
```suggestion
pip install django-vite inertia-django
```